Source/Icebreaker/Content/styles.css

```diff
@@ -28,172 +41,125 @@
         padding-top: 21px;
     }
 
-.bold-header {
-    font-family: 'Segoe UI';
-}
-
 body {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
     width: 100%;
     height: 100%;
-    margin: 0;
+    margin: 0rem;
     flex-direction: column;
 }
 
-/*Dark Theme rules*/
-.theme-dark .bold-header {
+.bold-header {
     text-align: left;
```

Comment:
Is this needed?

Reply:
Yes this is needed (if you are referring to the .bold-header css class)

Reply:
I meant the text-align: left;

Reply:
I put that from the redlines actually